core-bundle/src/Resources/contao/controllers/FrontendIndex.php

```diff
@@ -61,30 +61,9 @@ public function renderPage(PageModel $pageModel)
 		}
 
 		// Trigger the 404 page if an item is required but not given (see #8361)
-		if ($objPage->requireItem)
+		if ($objPage->requireItem && !isset($_GET['auto_item']))
```

Comment:
I would revert these changes, they become obsolete (and will cause a merge conflict) with https://github.com/contao/contao/pull/4304